build.gradle

```diff
@@ -1,10 +1,13 @@
 plugins {
     id "java"
-    id "edu.wpi.first.GradleRIO" version "2023.4.3"
+    id "edu.wpi.first.GradleRIO" version "2024.1.1-beta-4"
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_17
+    targetCompatibility = JavaVersion.VERSION_17
```

Comment:
These got put into a `java { } ` block cuz Future Gradle will want it that way (https://docs.gradle.org/8.5/userguide/upgrading_version_8.html#java_convention_deprecation).